functions.php

```diff
@@ -144,23 +146,23 @@ function twentyten_judge_child_wp_trim_excerpt( $text = '' ) {
 
 function judge_child_customizer( WP_Customize_Manager $wp_customize ) {
 	$wp_customize->add_setting( 'show_excerpts', array(
-		'default' => 'excerpts',
+		'default'   => 'excerpts',
 		'transport' => 'refresh',
 	) );
 
 	$wp_customize->add_section( 'judge_child_customizer', array(
-		'title' => __( 'Content Options', 'twentyten-judge' ),
+		'title'    => __( 'Content Options', 'twentyten-judge' ),
 		'priority' => 100,
 	) );
 
 	$wp_customize->add_control( 'judge_child_excerpt_control', array(
-		'label' => __( 'Content Excerpts' ),
-		'section' => 'judge_child_customizer',
+		'label'    => __( 'Content Excerpts' ),
+		'section'  => 'judge_child_customizer',
 		'settings' => 'show_excerpts',
-		'type' => 'radio',
-		'choices' => array(
+		'type'     => 'radio',
+		'choices'  => array(
 			'excerpts' => __( 'Only Excerpts', 'twentyten-judge' ),
-			'full' => __( 'Full Posts', 'twentyten-judge' )
+			'full'     => __( 'Full Posts', 'twentyten-judge' )
```

Comment:
In fact I would discourage this alignment in general. It ruins the blame of a file as soon as a longer element gets added as all the previous lines will have to be changed.

Secondly most IDE's will destroy this alignment anyway as soon as you use a autoformat feature.